activesupport/lib/active_support/number_helper/number_to_phone_converter.rb

```diff
@@ -10,26 +10,19 @@ def convert
       private
 
         def convert_to_phone_number(number)
-          if opts[:area_code]
-            convert_with_area_code(number)
-          else
-            convert_without_area_code(number)
+          parts = parse(number)
+          if parts
+            number = opts[:area_code] && !parts[0].blank? ? "(#{parts.shift}) " : ""
```

Comment:
This would create 2 almost identical methods, so I don't think that this would be good.  At least not if we want this code to be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)